packages/components/CHANGELOG.md

```diff
@@ -4,16 +4,20 @@
 
 ### Enhancements
 
-- Update `BorderControl` and `BorderBoxControl` to allow the passing of custom class names to popovers ([#39753](https://github.com/WordPress/gutenberg/pull/39753)).
+-   Update `BorderControl` and `BorderBoxControl` to allow the passing of custom class names to popovers ([#39753](https://github.com/WordPress/gutenberg/pull/39753)).
 
 ### Internal
 
 -   `BaseControl`: Convert to TypeScript ([#39468](https://github.com/WordPress/gutenberg/pull/39468)).
 
 ### New Features
 
-- Add `BorderControl` component ([#37769](https://github.com/WordPress/gutenberg/pull/37769)).
-- Add `BorderBoxControl` component ([#38876](https://github.com/WordPress/gutenberg/pull/38876)).
+-   Add `BorderControl` component ([#37769](https://github.com/WordPress/gutenberg/pull/37769)).
+-   Add `BorderBoxControl` component ([#38876](https://github.com/WordPress/gutenberg/pull/38876)).
+
```

Comment:
Changes on these lines are only about aligning the white spacing at the beginning of every list item